src/ng-currency-settings.provider.js

```diff
@@ -0,0 +1,31 @@
+export default function ngCurrencySettings() {
+  this.config = {
+    defaultFraction: 2,
+    defaultHardCap: false,
+    defaultMin: undefined,
+    defaultMax: undefined,
+    defaultCurrencySymbol: undefined
+  };
+
+  this.setDefaultCurrencySymbol = defaultCurrencySymbol => {
+    this.config.defaultCurrencySymbol = defaultCurrencySymbol;
+  };
+
+  this.setDefaultFraction = defaultFraction => {
+    this.config.defaultFraction = defaultFraction;
+  };
+
+  this.setDefaultHardCap = defaultHardCap => {
+    this.config.defaultHardCap = defaultHardCap;
+  };
+
+  this.setDefaultMin = defaultMin => {
+    this.config.defaultMin = defaultMin;
+  };
+
+  this.setDefaultMax = defaultMax => {
+    this.config.defaultMax = defaultMax;
+  };
+
+  this.$get = () => this.config;
```

Comment:
Rather then returning the whole object and allowing them to change the `defaults` on the running application, we should probably only allow them to get the values at this point.

```js
class $get {
  get currencySymbol() {
    return provider.config.currencySymbol;
  }
  // ...
}
```